src/js/pages/Service.js

```diff
@@ -34,19 +35,22 @@ const i18nMessages = defineMessages({
 
 const Service = ({ service, intl }) => {
   const {
-    image, title, slug, topic, steps, dynamicContent, additionalContent,
+    image, title, slug, topic, topic: {theme}, steps, dynamicContent, additionalContent,
     contacts, related
   } = service;
 
-  const { theme } = topic;
-
   //TODO: data below should be sourced as above
   const { services311 } = jsonFileData;
 
   //TODO: clean data where sourced
   const contact = cleanContacts(contacts)[0];
   const cleanedRelated = cleanRelatedServiceLinks(related);
 
+  //TODO: mapblock data should include contact data when sent via joplin
```

Comment:
currently joplin doesn't send us contact info with our map blocks so I'm fudging it.